scripts/hid_configurator/configurator_cli.py

```diff
@@ -169,14 +183,15 @@ def configurator():
     if not dev:
         return
 
-    if args.command == 'dfu':
-        perform_dfu(dev, args)
-    elif args.command == 'fwinfo':
-        perform_fwinfo(dev, args)
-    elif args.command == 'fwreboot':
-        perform_fwreboot(dev, args)
-    elif args.command == 'config':
-        perform_config(dev, args)
+    configurator.ALLOWED_COMMANDS[args.command](dev, args)
+
+configurator.ALLOWED_COMMANDS = {
+    'dfu' : perform_dfu,
```

Comment:
Not best solution, because names of cmd line params should be defined once to avoid situation that one has changed it in one plcae and forgot to do the same here. But do not change it, let it be as is.

Reply:
ok